shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/Delegation/Certificates.hs

```diff
@@ -124,12 +122,32 @@ isRetirePool _ = False
 
 newtype PoolDistr crypto = PoolDistr
   { unPoolDistr ::
-      ( Map
-          (KeyHash 'StakePool crypto)
-          (Rational, Hash crypto (VerKeyVRF crypto))
-      )
+      Map (KeyHash 'StakePool crypto) (PoolStake crypto)
   }
-  deriving (Show, Eq, ToCBOR, FromCBOR, NFData, NoUnexpectedThunks, Relation)
+  deriving stock (Show, Eq)
+  deriving newtype (ToCBOR, FromCBOR, NFData, NoUnexpectedThunks, Relation)
+
+data PoolStake crypto = PoolStake
+  { poolStakeStake :: !Rational,
+    poolStakeVrf :: !(Hash crypto (VerKeyVRF crypto))
```

Comment:
I realise it's unrelated to this PR, but now that we have this as a datatype it would be nice to comment explaining why we have the VRF in something that seems related only to stake :-)

Reply:
If you provide a comment, I'll add it :slightly_smiling_face: